Xamarin.Forms.Platform.Android/AndroidTicker.cs

```diff
@@ -53,10 +53,10 @@ static bool IsAnimatorEnabled()
 			var resolver = global::Android.App.Application.Context?.ContentResolver;
 			if (resolver == null)
 			{
-				return false;
+				return true;
 			}
 
-			var scale = global::Android.Provider.Settings.Global.GetFloat(resolver, global::Android.Provider.Settings.Global.AnimatorDurationScale, 0);
+			var scale = global::Android.Provider.Settings.Global.GetFloat(resolver, global::Android.Provider.Settings.Global.AnimatorDurationScale, 1);
```

Comment:
Before JellyBean (Api Level 17) we had the ANIMATOR_DURATION_SCALE property https://developer.android.com/reference/android/provider/Settings.System#ANIMATOR_DURATION_SCALE
Scaling factor for Animator-based animations. This affects both the start delay and duration of all such animations. Setting to 0 will cause animations to end immediately. The default value is 1.
From Api Level 17 or higher must use https://developer.android.com/reference/android/provider/Settings.Global.html#ANIMATOR_DURATION_SCALE

Depending on the Api Level, we could get an error (not found property). For that reason I suggest some changes.